## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 469291`
Commit: `c9f87b89cd079a3fd249ba1bcc3c429c32d83d05`

---
### `x86_64-linux`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python312Packages.mim-solvers</li>
    <li>python312Packages.proxsuite</li>
    <li>python312Packages.proxsuite.doc</li>
    <li>python312Packages.qpsolvers</li>
    <li>python312Packages.qpsolvers.dist</li>
    <li>python313Packages.mim-solvers</li>
    <li>python313Packages.proxsuite</li>
    <li>python313Packages.proxsuite.doc</li>
    <li>python313Packages.qpsolvers</li>
    <li>python313Packages.qpsolvers.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 423 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>atopile (python313Packages.atopile)</li>
    <li>atopile.dist (python313Packages.atopile.dist)</li>
    <li>babeldoc</li>
    <li>babeldoc.dist</li>
    <li>beets (python313Packages.beets)</li>
    <li>beets.dist (python313Packages.beets.dist)</li>
    <li>beets.doc (python313Packages.beets.doc)</li>
    <li>beets.man (python313Packages.beets.man)</li>
    <li>calibre</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>dolfinx</li>
    <li>exo</li>
    <li>exo.dist</li>
    <li>gemmi (python313Packages.gemmi)</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>llama-cpp-rocm</li>
    <li>magma (magma-hip)</li>
    <li>magma.test (magma-hip.test)</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>ollama-rocm</li>
    <li>onnx</li>
    <li>onnx.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>pianotrans</li>
    <li>pianotrans.dist</li>
    <li>piper-tts</li>
    <li>piper-tts.dist</li>
    <li>pitivi</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.atomman</li>
    <li>python312Packages.atomman.dist</li>
    <li>python312Packages.beetcamp</li>
    <li>python312Packages.beetcamp.dist</li>
    <li>python312Packages.beets</li>
    <li>python312Packages.beets-audible</li>
    <li>python312Packages.beets-audible.dist</li>
    <li>python312Packages.beets.dist</li>
    <li>python312Packages.beets.doc</li>
    <li>python312Packages.beets.man</li>
    <li>python312Packages.boost-histogram</li>
    <li>python312Packages.boost-histogram.dist</li>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
    <li>python312Packages.clu</li>
    <li>python312Packages.clu.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.deepface</li>
    <li>python312Packages.deepface.dist</li>
    <li>python312Packages.dm-sonnet</li>
    <li>python312Packages.dm-sonnet.dist</li>
    <li>python312Packages.ducc0</li>
    <li>python312Packages.ducc0.dist</li>
    <li>python312Packages.edward</li>
    <li>python312Packages.edward.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.fenics-basix</li>
    <li>python312Packages.fenics-basix.dist</li>
    <li>python312Packages.fenics-dolfinx</li>
    <li>python312Packages.fenics-dolfinx.dist</li>
    <li>python312Packages.fenics-ffcx</li>
    <li>python312Packages.fenics-ffcx.dist</li>
    <li>python312Packages.firedrake</li>
    <li>python312Packages.firedrake.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.flaxlib</li>
    <li>python312Packages.flaxlib.dist</li>
    <li>python312Packages.freud</li>
    <li>python312Packages.freud.dist</li>
    <li>python312Packages.gemmi</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.hist</li>
    <li>python312Packages.hist.dist</li>
    <li>python312Packages.insightface</li>
    <li>python312Packages.insightface.dist</li>
    <li>python312Packages.islpy</li>
    <li>python312Packages.islpy.dist</li>
    <li>python312Packages.keras</li>
    <li>python312Packages.keras.dist</li>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.langchain-huggingface</li>
    <li>python312Packages.langchain-huggingface.dist</li>
    <li>python312Packages.lerobot</li>
    <li>python312Packages.lerobot.dist</li>
    <li>python312Packages.librosa</li>
    <li>python312Packages.librosa.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llm-sentence-transformers</li>
    <li>python312Packages.llm-sentence-transformers.dist</li>
    <li>python312Packages.loopy</li>
    <li>python312Packages.loopy.dist</li>
    <li>python312Packages.manifest-ml</li>
    <li>python312Packages.manifest-ml.dist</li>
    <li>python312Packages.manifold3d</li>
    <li>python312Packages.manifold3d.dist</li>
    <li>python312Packages.meeko</li>
    <li>python312Packages.meeko.dist</li>
    <li>python312Packages.mhcflurry</li>
    <li>python312Packages.mhcflurry.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.mistral-common</li>
    <li>python312Packages.mistral-common.dist</li>
    <li>python312Packages.mlcroissant</li>
    <li>python312Packages.mlcroissant.dist</li>
    <li>python312Packages.mmcv</li>
    <li>python312Packages.mmcv.dist</li>
    <li>python312Packages.mplhep</li>
    <li>python312Packages.mplhep.dist</li>
    <li>python312Packages.mtcnn</li>
    <li>python312Packages.mtcnn.dist</li>
    <li>python312Packages.nanobind</li>
    <li>python312Packages.nanobind.dist</li>
    <li>python312Packages.nanoeigenpy</li>
    <li>python312Packages.nanoeigenpy.dev</li>
    <li>python312Packages.nanoeigenpy.doc</li>
    <li>python312Packages.nifty8</li>
    <li>python312Packages.nifty8.dist</li>
    <li>python312Packages.onnx</li>
    <li>python312Packages.onnx.dist</li>
    <li>python312Packages.onnxconverter-common</li>
    <li>python312Packages.onnxconverter-common.dist</li>
    <li>python312Packages.onnxmltools</li>
    <li>python312Packages.onnxmltools.dist</li>
    <li>python312Packages.onnxruntime-tools</li>
    <li>python312Packages.onnxruntime-tools.dist</li>
    <li>python312Packages.onnxslim</li>
    <li>python312Packages.onnxslim.dist</li>
    <li>python312Packages.paddle2onnx</li>
    <li>python312Packages.paddle2onnx.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.phonopy</li>
    <li>python312Packages.phonopy.dist</li>
    <li>python312Packages.piano-transcription-inference</li>
    <li>python312Packages.piano-transcription-inference.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pyopencl</li>
    <li>python312Packages.pyopencl.dist</li>
    <li>python312Packages.pytorch-pfn-extras</li>
    <li>python312Packages.pytorch-pfn-extras.dist</li>
    <li>python312Packages.reikna</li>
    <li>python312Packages.reikna.dist</li>
    <li>python312Packages.retinaface</li>
    <li>python312Packages.retinaface.dist</li>
    <li>python312Packages.sentence-transformers</li>
    <li>python312Packages.sentence-transformers.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skl2onnx</li>
    <li>python312Packages.skl2onnx.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.soxr</li>
    <li>python312Packages.soxr.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.tensorflow-datasets</li>
    <li>python312Packages.tensorflow-datasets.dist</li>
    <li>python312Packages.tf2onnx</li>
    <li>python312Packages.tf2onnx.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.tinygrad</li>
    <li>python312Packages.tinygrad.dist</li>
    <li>python312Packages.torch-audiomentations</li>
    <li>python312Packages.torch-audiomentations.dist</li>
    <li>python312Packages.torchWithRocm</li>
    <li>python312Packages.torchWithRocm.cxxdev</li>
    <li>python312Packages.torchWithRocm.dev</li>
    <li>python312Packages.torchWithRocm.dist</li>
    <li>python312Packages.torchWithRocm.lib</li>
    <li>python312Packages.torchcrepe</li>
    <li>python312Packages.torchcrepe.dist</li>
    <li>python312Packages.torchlibrosa</li>
    <li>python312Packages.torchlibrosa.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.ultralytics</li>
    <li>python312Packages.ultralytics.dist</li>
    <li>python312Packages.units-llnl</li>
    <li>python312Packages.units-llnl.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>python312Packages.unstructured-inference</li>
    <li>python312Packages.unstructured-inference.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python312Packages.vmas</li>
    <li>python312Packages.vmas.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python312Packages.xgrammar</li>
    <li>python312Packages.xgrammar.dist</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.atomman</li>
    <li>python313Packages.atomman.dist</li>
    <li>python313Packages.beetcamp</li>
    <li>python313Packages.beetcamp.dist</li>
    <li>python313Packages.beets-audible</li>
    <li>python313Packages.beets-audible.dist</li>
    <li>python313Packages.boost-histogram</li>
    <li>python313Packages.boost-histogram.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.clu.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.ducc0</li>
    <li>python313Packages.ducc0.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.fenics-basix</li>
    <li>python313Packages.fenics-basix.dist</li>
    <li>python313Packages.fenics-dolfinx</li>
    <li>python313Packages.fenics-dolfinx.dist</li>
    <li>python313Packages.fenics-ffcx</li>
    <li>python313Packages.fenics-ffcx.dist</li>
    <li>python313Packages.firedrake</li>
    <li>python313Packages.firedrake.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flaxlib</li>
    <li>python313Packages.flaxlib.dist</li>
    <li>python313Packages.freud</li>
    <li>python313Packages.freud.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.hist</li>
    <li>python313Packages.hist.dist</li>
    <li>python313Packages.insightface</li>
    <li>python313Packages.insightface.dist</li>
    <li>python313Packages.islpy</li>
    <li>python313Packages.islpy.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.librosa</li>
    <li>python313Packages.librosa.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.loopy</li>
    <li>python313Packages.loopy.dist</li>
    <li>python313Packages.manifest-ml</li>
    <li>python313Packages.manifest-ml.dist</li>
    <li>python313Packages.manifold3d</li>
    <li>python313Packages.manifold3d.dist</li>
    <li>python313Packages.meeko</li>
    <li>python313Packages.meeko.dist</li>
    <li>python313Packages.mhcflurry</li>
    <li>python313Packages.mhcflurry.dist</li>
    <li>python313Packages.mistral-common</li>
    <li>python313Packages.mistral-common.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mplhep</li>
    <li>python313Packages.mplhep.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.nanobind</li>
    <li>python313Packages.nanobind.dist</li>
    <li>python313Packages.nanoeigenpy</li>
    <li>python313Packages.nanoeigenpy.dev</li>
    <li>python313Packages.nanoeigenpy.doc</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.onnx</li>
    <li>python313Packages.onnx.dist</li>
    <li>python313Packages.onnxconverter-common</li>
    <li>python313Packages.onnxconverter-common.dist</li>
    <li>python313Packages.onnxmltools</li>
    <li>python313Packages.onnxmltools.dist</li>
    <li>python313Packages.onnxruntime-tools</li>
    <li>python313Packages.onnxruntime-tools.dist</li>
    <li>python313Packages.onnxslim</li>
    <li>python313Packages.onnxslim.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.phonopy</li>
    <li>python313Packages.phonopy.dist</li>
    <li>python313Packages.piano-transcription-inference</li>
    <li>python313Packages.piano-transcription-inference.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyopencl</li>
    <li>python313Packages.pyopencl.dist</li>
    <li>python313Packages.pytorch-pfn-extras</li>
    <li>python313Packages.pytorch-pfn-extras.dist</li>
    <li>python313Packages.reikna</li>
    <li>python313Packages.reikna.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.sentence-transformers</li>
    <li>python313Packages.sentence-transformers.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skl2onnx</li>
    <li>python313Packages.skl2onnx.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.soxr</li>
    <li>python313Packages.soxr.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-datasets.dist</li>
    <li>python313Packages.tf2onnx</li>
    <li>python313Packages.tf2onnx.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.torch-audiomentations</li>
    <li>python313Packages.torch-audiomentations.dist</li>
    <li>python313Packages.torchWithRocm</li>
    <li>python313Packages.torchWithRocm.cxxdev</li>
    <li>python313Packages.torchWithRocm.dev</li>
    <li>python313Packages.torchWithRocm.dist</li>
    <li>python313Packages.torchWithRocm.lib</li>
    <li>python313Packages.torchcrepe</li>
    <li>python313Packages.torchcrepe.dist</li>
    <li>python313Packages.torchlibrosa</li>
    <li>python313Packages.torchlibrosa.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.tyro</li>
    <li>python313Packages.tyro.dist</li>
    <li>python313Packages.ultralytics</li>
    <li>python313Packages.ultralytics.dist</li>
    <li>python313Packages.units-llnl</li>
    <li>python313Packages.units-llnl.dist</li>
    <li>python313Packages.unsloth</li>
    <li>python313Packages.unsloth-zoo</li>
    <li>python313Packages.unsloth-zoo.dist</li>
    <li>python313Packages.unsloth.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>python313Packages.xgrammar</li>
    <li>python313Packages.xgrammar.dist</li>
    <li>rocmPackages.hipblas</li>
    <li>rocmPackages.hipblaslt</li>
    <li>rocmPackages.hipblaslt.benchmark</li>
    <li>rocmPackages.hipsolver</li>
    <li>rocmPackages.migraphx</li>
    <li>rocmPackages.miopen (rocmPackages.miopen-hip)</li>
    <li>rocmPackages.mivisionx (rocmPackages.mivisionx-hip)</li>
    <li>rocmPackages.rocalution</li>
    <li>rocmPackages.rocblas</li>
    <li>rocmPackages.rocblas.benchmark</li>
    <li>rocmPackages.rocblas.test</li>
    <li>rocmPackages.rocsolver</li>
    <li>supergee</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>unbook</li>
    <li>unstructured-api</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
    <li>voicevox</li>
    <li>voicevox-engine</li>
    <li>voicevox-engine.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wyoming-piper</li>
    <li>wyoming-piper.dist</li>
    <li>zluda</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.proxsuite</summary>
<pre>-- Performing Test res_-Wwrite-strings - Success
-- Performing Test res_-Wconversion
-- Performing Test res_-Wconversion - Success
-- Found Doxygen: /nix/store/wcain233jcs4jk9dgydcm8rkr7x0b5g0-doxygen-1.15.0/bin/doxygen (found version &quot;1.15.0&quot;) found components: doxygen dot
-- MathJax version 3 will be used. If MATHJAX_RELPATH is unset we will use the vendored MathJax. If you have set it, check that it&#x27;s pointing at a MathJax 3 distribution.
-- CMAKE_SYSTEM_PROCESSOR: x86_64
-- [Proxsuite] Using C++ standard: 17
-- Found Simde: /nix/store/ivxh051vqgqj3v3355lffd7j8z4pl1wi-simde-0.8.2/include
-- Found Python3: /nix/store/fdibxyh7xcmqrc172y78awzhxs292gq1-python3-3.12.12/bin/python (found version &quot;3.12.12&quot;) found components: Interpreter Development.Module
-- PythonLibraryDirs: 
-- PythonLibVersionString: 3.12.12
-- Python site lib: lib/python3.12/site-packages
-- Python include dirs: /nix/store/fdibxyh7xcmqrc172y78awzhxs292gq1-python3-3.12.12/include/python3.12
CMake Error at /nix/store/z2ks0xrkvqq177015gh7f7i8r7dqrdzq-python3.12-nanobind-2.10.1/lib/python3.12/site-packages/nanobind/cmake/nanobind-config.cmake:8 (message):
  nanobind requires Python 3.9 or newer (found Python ).
Call Stack (most recent call first):
  bindings/python/CMakeLists.txt:34 (find_package)


-- Configuring incomplete, errors occurred!</pre>
</details>
<details>
<summary>python313Packages.proxsuite</summary>
<pre>-- Performing Test res_-Wwrite-strings - Success
-- Performing Test res_-Wconversion
-- Performing Test res_-Wconversion - Success
-- Found Doxygen: /nix/store/wcain233jcs4jk9dgydcm8rkr7x0b5g0-doxygen-1.15.0/bin/doxygen (found version &quot;1.15.0&quot;) found components: doxygen dot
-- MathJax version 3 will be used. If MATHJAX_RELPATH is unset we will use the vendored MathJax. If you have set it, check that it&#x27;s pointing at a MathJax 3 distribution.
-- CMAKE_SYSTEM_PROCESSOR: x86_64
-- [Proxsuite] Using C++ standard: 17
-- Found Simde: /nix/store/ivxh051vqgqj3v3355lffd7j8z4pl1wi-simde-0.8.2/include
-- Found Python3: /nix/store/3lll9y925zz9393sa59h653xik66srjb-python3-3.13.9/bin/python (found version &quot;3.13.9&quot;) found components: Interpreter Development.Module
-- PythonLibraryDirs: 
-- PythonLibVersionString: 3.13.9
-- Python site lib: lib/python3.13/site-packages
-- Python include dirs: /nix/store/3lll9y925zz9393sa59h653xik66srjb-python3-3.13.9/include/python3.13
CMake Error at /nix/store/248picqmpbbwmldb6ylg0mgxwgdwph5w-python3.13-nanobind-2.10.1/lib/python3.13/site-packages/nanobind/cmake/nanobind-config.cmake:8 (message):
  nanobind requires Python 3.9 or newer (found Python ).
Call Stack (most recent call first):
  bindings/python/CMakeLists.txt:34 (find_package)


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

--- a/pkgs/by-name/pr/proxsuite/fix-cmake-python-version.patch
+++ b/pkgs/by-name/pr/proxsuite/fix-cmake-python-version.patch
@@ -1,0 +1,14 @@
+diff --git a/bindings/python/CMakeLists.txt b/bindings/python/CMakeLists.txt
+index 07678691f..1d85e5eb2 100644
+--- a/bindings/python/CMakeLists.txt
++++ b/bindings/python/CMakeLists.txt
+@@ -31,6 +31,9 @@ execute_process(
+   OUTPUT_STRIP_TRAILING_WHITESPACE
+   OUTPUT_VARIABLE nanobind_ROOT
+ )
++if(NOT Python_VERSION AND Python3_VERSION)
++  set(Python_VERSION ${Python3_VERSION})
++endif()
+ find_package(nanobind CONFIG)
+ cmake_policy(POP)
+ if(NOT nanobind_FOUND)

--- a/pkgs/by-name/pr/proxsuite/package.nix
+++ b/pkgs/by-name/pr/proxsuite/package.nix
@@ -36,6 +36,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-1+a5tFOlEwzhGZtll35EMFceD0iUOOQCbwJd9NcFDlk=";
   };
 
+  patches = [
+    # Set Python_VERSION to Python3_VERSION if not already set
+    ./fix-cmake-python-version.patch
+  ];
+
   # ref. https://github.com/Simple-Robotics/proxsuite/pull/408 merged upstream
   postPatch = ''
     substituteInPlace CMakeLists.txt --replace-fail \

--- a/pkgs/development/python-modules/nanobind/default.nix
+++ b/pkgs/development/python-modules/nanobind/default.nix
@@ -26,7 +26,7 @@
 }:
 buildPythonPackage rec {
   pname = "nanobind";
-  version = "2.9.2";
+  version = "2.10.1";
   pyproject = true;
 
   src = fetchFromGitHub {
@@ -34,7 +34,7 @@ buildPythonPackage rec {
     repo = "nanobind";
     tag = "v${version}";
     fetchSubmodules = true;
-    hash = "sha256-cC+sf2FUm1jdGMRdDoaQK8rjUVkWjn/53c1HQ5gsUWs=";
+    hash = "sha256-LNr3d0ytnkcJOps6DzdZtyiihFwNgv34s5RJsRBD844=";
   };
 
   build-system = [

